lib/tasks/scripts/metabase.rb

```
@@ -0,0 +1,106 @@
+# This script is based on https://github.com/itmi-id/metabase-migration
+# Since this uses the metabase API which can change over time, it's a good idea
+# to test run this before running against production.
+#
+# You'll need a source question, and a collection in the destination env
+# where the question should be copied to.
+#
+# 1. Set credentials in your local .env
+# METABASE_SOURCE_HOST=metabase-sandbox.simple.org
+# METABASE_SOURCE_USERNAME=prabhanshu@nilenso.com
+# METABASE_SOURCE_PASSWORD=password
+# METABASE_DESTINATION_HOST=metabase.simple.org
+# METABASE_DESTINATION_USERNAME=prabhanshu@nilenso.com
+# METABASE_DESTINATION_PASSWORD=password
+#
+# Hostnames shouldn't have the http:// or https:// prefix.
+#
+# 2. Run commands in a REPL
+# Metabase.new
+# instance.duplicate_question(90, 23, 2)
+# instance.duplicate_question(<question_id>, <destination_collection_id>, <destination_database_id>)
+#
+# The database_id can be found from the URL when you choose
+# the DB from https://metabase-sandbox.simple.org/browse
+
+require "net/http"
+
+class Metabase
+  def initialize(source_host = ENV["METABASE_SOURCE_HOST"],
+    source_username = ENV["METABASE_SOURCE_USERNAME"],
+    source_password = ENV["METABASE_SOURCE_PASSWORD"],
+    destination_host = ENV["METABASE_DESTINATION_HOST"],
+    destination_username = ENV["METABASE_DESTINATION_USERNAME"],
+    destination_password = ENV["METABASE_DESTINATION_PASSWORD"])
```

can you `fetch` on these so it will fail fast if someone misses one of these?